src/components/text/UnderlinedText.tsx

```diff
@@ -0,0 +1,36 @@
9 9.82587 30.8324 9.87979 29.716 9.89207C29.2694 9.89698 28.748 9.86025 28.3014 9.86517C26.7385 9.88236 25.2502 9.91996 23.6872 9.93716C23.3895 9.94043 23.0918 9.94371 22.9441 10.0515C22.9446 10.094 22.5725 10.0981 22.3494 10.1217C19.1493 10.1782 15.9492 10.2346 12.7491 10.291C11.4839 10.305 10.2933 10.3393 9.0283 10.3744C8.3587 10.403 7.76399 10.4733 7.09416 10.4807C5.75425 10.4742 4.41434 10.4677 3.0742 10.4399C2.62741 10.4236 2.25435 10.3428 1.88105 10.2407C1.05979 10.0162 0.461347 9.74675 0.383184 9.40787C0.305722 9.13269 0.153599 8.83709 0.0761365 8.56191C0.0747352 8.43452 0.147761 8.30632 0.220786 8.17811C0.294278 8.09237 0.293344 8.00745 0.366836 7.92171C0.511485 7.53791 0.432621 7.13534 0.726122 6.74991C0.871939 6.47227 1.01775 6.19463 0.865866 5.92027C0.865165 5.85657 1.08774 5.79042 1.16123 5.70468C1.4552 5.36171 1.67451 4.99833 1.96848 4.65536C2.11523 4.46264 2.48503 4.24624 2.33431 4.07803C2.10776 3.78326 2.84898 3.49907 2.39939 3.22798C2.2496 3.14469 2.47148 3.01485 2.61916 2.90706C2.6187 2.8646 2.84128 2.79845 2.76685 2.79926C2.31889 2.67679 2.76405 2.54449 2.91197 2.45793C3.2078 2.28481 2.908 2.09701 3.57596 1.91979C4.09555 1.78667 4.5414 1.71807 5.13681 1.71152C6.77396 1.67228 8.48553 1.63222 10.1229 1.6142C12.8764 1.56268 15.5556 1.51198 18.3093 1.48168C21.3606 1.42689 24.4865 1.3925 27.5377 1.3377C29.6216 1.31478 31.7056 1.29185 33.7895 1.26893C36.4689 1.23945 39.0738 1.2108 41.7531 1.18133C43.0184 1.16741 44.2092 1.15431 45.4747 1.16162C47.0376 1.14443 48.5261 1.12805 50.0893 1.13209C52.3965 1.10671 54.6296 1.10338 56.937 1.09923C59.2445 1.09508 61.6263 1.09012 63.9338 1.08597C66.0924 1.08345 68.1763 1.06053 70.3351 1.07925C73.2382 1.08978 76.1413 1.10031 79.0442 1.08961C81.0539 1.08874 83.0629 1.02417 85.0727 1.0233C87.2313 1.02079 89.4645 1.03869 91.6229 1.01494C94.1534 0.987108 96.6837 0.938041 99.2886 0.909386C103.456 0.863538 107.625 0.83892 111.718 0.793891C114.695 0.761143 117.672 0.707164 120.723 0.652366C124.221 0.592655 127.793 0.532125 131.291 0.472415C134.342 0.417616 137.319 0.321175 140.295 0.245965C142.676 0.177304 145.132 0.129055 147.588 0.0808067C148.109 0.0750758 148.556 0.133856 149.078 0.170587C148.857 0.427815 148.933 0.596844 149.307 0.720136C148.713 0.83284 148.937 0.936538 149.31 0.996138C149.088 1.12598 148.866 1.23459 148.793 1.34156C148.645 1.47059 148.721 1.59715 149.391 1.58978C149.02 1.65757 148.722 1.70331 148.276 1.77191C148.351 1.81356 148.426 1.8552 148.501 1.87561C149.097 1.95399 149.473 2.26836 149.028 2.42188C148.88 2.48722 148.433 2.49213 148.136 2.51663C148.136 2.53786 148.136 2.58033 148.136 2.58033C148.807 2.65788 148.66 2.8506 148.737 2.99839C148.738 3.16824 148.591 3.29726 148.22 3.44997C148.072 3.51531 148.223 3.64105 148.149 3.74803C148.15 3.81172 148.225 3.89582 148.151 3.9391C147.485 4.22248 147.563 4.54012 147.492 4.8594C147.493 4.94433 147.419 5.00884 147.345 5.07335C147.271 5.13786 146.752 5.27098 147.274 5.41386Z' fill='${encodeURIComponent(brandPrimary)}'/%3E%3C/svg%3E%0A")`,
```

Comment:
There is [an issue](https://github.com/chakra-ui/chakra-ui/issues/7548) in chakra-ui that results in setting `url-gradient` property instead of `url` when `bgImage` is used directly